src/model/Model.ts

```diff
@@ -332,7 +348,12 @@ export class Model {
    */
   protected $fillField(key: string, attr: Attribute, value: any): void {
     if (value !== undefined) {
-      this[key] = attr.make(value)
+      if (attr instanceof MorphTo) {
+        this[key] = attr.make(value, this[attr.$getType()])
+      } else {
+        this[key] = attr.make(value)
+      }
```

Comment:
We need the type to be provided in order to make the child entity in inverse polymorphic relations.